operator/builtin/input/file/config_test.go

```diff
@@ -749,6 +750,9 @@ func TestMapStructureDecodeConfig(t *testing.T) {
 		"max_log_size":         1024 * 1024,
 		"max_concurrent_files": 1024,
 		"encoding":             "utf16",
+		"force_flush_period": map[string]interface{}{
+			"Duration": time.Millisecond * 500,
+		},
```

Comment:
For the sake of consistency, let's make sure all duration parameters are instantiated the same way in this method. No reason to have `poll_interval` and `force_flush_period` instantiated differently.